.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -5,13 +5,6 @@ title: "(short issue description)"
 labels: [feature-request, needs-triage]
 assignees: []
 body:
-  - type: markdown 
-    attributes: 
-      value: |
-        ## Community Note
-        1. Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
-        2. Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
-        3. If you are interested in working on this issue, please leave a comment
```

Comment:
Trivial: crt-swift is in dev preview, we might want to keep this.

Reply:
readded